### ISSUE TITLE
Staging Review finding: Unclear language in link text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#92191]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Sitewide Content, Accessibility, and Information Architecture
**Product:** National tests and prep courses benefit detail page
**Feature:** 

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @kkitagawa-bosch (content)

### Description

Both the "National tests and prep courses" page as well as the "Licensing and certification tests and prep courses" page contain a link labeled "Check the rates by benefit". This link text is unclear; I'm not sure what "the rates" refers to. 

## Recommended action

I would recommend adding additional information to the link text to clarify 

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Content principles](https://design.va.gov/content-style-guide/content-principles)
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bethpottsVADEPO** [comment]
We ended up removing this link altogether as I mentioned in staging review.
